x/pool-incentives/keeper/keeper.go

```diff
@@ -100,29 +100,23 @@ func (k Keeper) CreateConcentratedLiquidityPoolGauge(ctx sdk.Context, poolId uin
 		return fmt.Errorf("pool %d is not concentrated liquidity pool", poolId)
 	}
 
-	gaugeIdCL, err := k.incentivesKeeper.CreateGauge(
+	_, err = k.incentivesKeeper.CreateGauge(
 		ctx,
 		true,
 		k.accountKeeper.GetModuleAddress(types.ModuleName),
 		sdk.Coins{},
-		// dummy variable so that the existing logic does not break
-		// CreateGauge checks if LockQueryType is `ByDuration` or not, we bypass this check by passing
-		// lockQueryType as byTime. Although we do not need this check, we still cannot pass empty struct.
 		lockuptypes.QueryCondition{
-			LockQueryType: lockuptypes.ByTime,
-			Denom:         appParams.BaseCoinUnit,
+			LockQueryType: lockuptypes.NoLock,
+			Denom:         incentivestypes.NoLockInternalGaugeDenom(pool.GetId()),
 		},
 		ctx.BlockTime(),
 		1,
+		poolId,
 	)
 	if err != nil {
 		return err
 	}
 
-	incParams := k.incentivesKeeper.GetEpochInfo(ctx)
-	// lockable duration is epoch duration because we create incentive_record on every epoch
-	k.SetPoolGaugeId(ctx, poolId, incParams.Duration, gaugeIdCL)
```

Comment:
Note: this must now be set internally in `CreateGauge` specifically for `NoLock` gauges